### PR TITLE
add script append-note-from-clipboard

### PR DESCRIPTION
**Pull request by ayoub-g**
## Description

This script appends clipboard content to an existing note.

## Type of change
- [x] New script command

## Screenshot

https://user-images.githubusercontent.com/841163/130296709-e0fd8a02-4910-413c-87a1-38a549247cb6.mov

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/notes/append-note-from-clipboard.applescript [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# @raycast.title Append Note From Clipboard
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/notes.png
+# @raycast.argument1 { "type": "text", "placeholder": "Note Name" }
+# @raycast.packageName Notes
+
+# Documentation:
+# @raycast.description this script accepts an existent note name, appends to it the content of the clipboard
```
Comment (unnamedd) [inline commands/apps/notes/append-note-from-clipboard.applescript:14]: What do you think?
```suggestion
# @raycast.description Script to append to an existing note content from clipboard.
```

File: commands/apps/notes/append-note-from-clipboard.applescript [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Append Note From Clipboard
```
Comment (unnamedd) [inline commands/apps/notes/append-note-from-clipboard.applescript:5]:
```suggestion
# @raycast.title Append Content From Clipboard
```

**Review by unnamedd — CHANGES_REQUESTED** [review]

File: commands/apps/notes/append-note-from-clipboard.applescript (hunk 2 above)
Comment (ayoub-g) [reply commands/apps/notes/append-note-from-clipboard.applescript:5]: Yes it's accurate

File: commands/apps/notes/append-note-from-clipboard.applescript (hunk 1 above)
Comment (ayoub-g) [reply commands/apps/notes/append-note-from-clipboard.applescript:14]: Yes, it's accurate, thank you @unnamedd

**Review by unnamedd — APPROVED** [review]
Looks good to me! 🎉 

Thanks for your contribution @ayoub-g!

**Comment by mattisssa** [comment]
Wow, this is nice! 👍

**Pull request merged**